java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture5/tasks/_3/Number3.java

```diff
@@ -0,0 +1,46 @@
+package com.zoolatech.lecture5.tasks._3;
+
+/*
+Create a method that accepts a list of words, and displays only unique one (order can be random).
+Bonus task: treat strings in a different case as the same one: “Apple” and “apple” should be treated as the same string
+(don’t use any methods that create a new string object - like toUpperCase or toLowerCase - you can work only with a given input).
+Input: [“apple”, “orange”, “pear”, “apple”, “banana”, “orange”]
+Output: apple
+		orange
+		pear
+		banana
+ */
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Number3 {
+    public static void main(String[] args) {
+        List<String> stringList = new ArrayList<>();
```

Comment:
instead of `stringList` try to find a better naming for the variable - try to think what it holds